src/SFML/Graphics/Image.cpp

```diff
@@ -179,34 +182,25 @@ bool Image::loadFromFile(const std::filesystem::path& filename)
 
     if (ptr)
```

Comment:
Can we invert these conditions and reduce the nesting here since we're touching this function?